src/libstore/builtins/buildenv.cc

```diff
@@ -13,6 +13,24 @@ struct State
     unsigned long symlinks = 0;
 };
 
+class BuildEnvFileConflictError : public std::exception
```

Comment:
```suggestion
/** This is private to this file and always caught, so we don't worry about it rendering in a nice fancy way etc. */
class BuildEnvFileConflictError : public std::exception
```

Reply:
And maybe line wrap it :)

Reply:
I rewrote the comment a bit. Hope you like it ;)